tests/Cache/CacheClientTest.php

```diff
@@ -117,26 +116,40 @@ public function testCreateCacheEmptyName()
         $this->assertEquals(get_class($response) . ": {$response->message()}", "$response");
     }
 
-    public function testCreateCacheNullName() {
-        $this->expectException(\TypeError::class);
+    public function testCreateCacheNullName()
+    {
+        $this->expectException(TypeError::class);
         $this->client->createCache(null);
     }
 
-    public function testCreateCacheBadName() {
+    public function testCreateCacheBadName()
+    {
         $response = $this->client->createCache(1);
         $this->assertNotNull($response->asError());
         $this->assertEquals(MomentoErrorCode::INVALID_ARGUMENT_ERROR, $response->asError()->errorCode());
     }
 
-    public function testCreateCacheBadAuth() {
+    public function testCreateCacheBadAuth()
+    {
         $client = $this->getBadAuthTokenClient();
         $response = $client->createCache(uniqid());
         $this->assertNotNull($response->asError());
         $this->assertEquals(MomentoErrorCode::AUTHENTICATION_ERROR, $response->asError()->errorCode());
     }
 
+    private function getBadAuthTokenClient(): SimpleCacheClient
+    {
+        $badEnvName = "_MOMENTO_BAD_AUTH_TOKEN";
+        putenv("{$badEnvName}={$this->BAD_AUTH_TOKEN}");
+        $authProvider = new EnvMomentoTokenProvider($badEnvName);
+        putenv($badEnvName);
+        return new SimpleCacheClient($authProvider, $this->DEFAULT_TTL_SECONDS);
+    }
+
```

Comment:
Can you move this to the top of the class, please, either before `setUp()` or after `tearDown()`?